sdk/src/main/scala/com/horizen/utils/WithdrawalEpochUtils.scala

```diff
@@ -24,6 +24,6 @@ object WithdrawalEpochUtils {
   }
 
   def canSubmitCertificate(withdrawalEpochInfo: WithdrawalEpochInfo, params: NetworkParams): Boolean = {
-    (withdrawalEpochInfo.epoch > 0) && (withdrawalEpochInfo.lastEpochIndex < params.withdrawalEpochLength / 5)
+    (withdrawalEpochInfo.epoch > 0) && (withdrawalEpochInfo.lastEpochIndex <= params.withdrawalEpochLength / 5)
```

Comment:
I think, that check must be inclusive. `lastEpochIndex` starts from 0, but when first mc block ref data arrived == 1